src/sage/rings/number_field/S_unit_solver.py

```diff
@@ -1,24 +1,10 @@
 # sage.doctest: needs sage.rings.number_field sage.rings.padics
 r"""
-Solve S-unit equation x + y = 1
+Solve `S`-unit equation `x + y = 1`
```

Comment:
This title stands out grammatically. Perhaps "Solver for the ..."

Reply:
Thanks. Done.